src/Services/Page.vala

```diff
@@ -219,18 +219,55 @@ public class ENotes.PageTable : DatabaseTable {
         }
     }
 
-    private const string[] SYMBOLS_BLACKLIST = {"#", "`", "\t", "<br>", ">", "<", "\n", "~" };
+    // Describs member of BlackList used in cleanup. reg is the regular expression and replace the string it will be replaced
+    private struct BLMember {
+        public GLib.Regex reg;
+        public string replace;
+        
+        public BLMember (GLib.Regex reg, string replace) {
+            this.reg = reg;
+            this.replace = replace;
+        }
+    
+    }
+      
     private string cleanup (string line) {
+ 
+    // This list is used for complex commands mostly by plugins e.g. "" <youtube [Link]>
+       BLMember[] Regex_complex_commands;
```

Comment:
Check how to capitalize variable names https://elementary.io/docs/code/reference#variable-names-class-names-function-names